lib/handler/status/events.c

```diff
@@ -102,6 +103,7 @@ static h2o_iovec_t events_status_final(void *priv, h2o_globalconf_t *gconf, h2o_
                        " \"http2-errors.connect\": %" PRIu64 ", \n"
                        " \"http2-errors.enhance-your-calm\": %" PRIu64 ", \n"
                        " \"http2-errors.inadequate-security\": %" PRIu64 ", \n"
+                       " \"http2-errors.idle-timeout\": %" PRIu64 ", \n"
```

Comment:
I think this should be named "http2.idle-timeout", and be placed after "http2.write_closed", due to the following reasons:
* Doing so aligns the order with h2o_context_t and st_events_status_ctx_t.
* Closing a connection due to idle timeout is not an error. It is much closer to "read-close" or "write-close" that we have right below.

Reply:
Agreed. I feel `http2.` makes more sense. Fixed at bacaf828d75e9808e2e628bb484ed38e02492a0a

Reply:
@gfx Do you mind addressing the second point as well?

Reply:
Oops. I missed the second one. Fixed bacaf82